docs/content/docs/getting-started/nuxt.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Nuxt
+---
+
+# Nuxt
+
+This guide will walk through setting up your first workflow in a Nuxt app. Along the way, you'll learn more about the concepts that are fundamental to using the development kit in your own projects.
+
+---
+
+<Steps>
+
+<Step>
+## Create Your Nuxt Project
+
+Start by creating a new Nuxt project. This command will create a new directory named `nuxt-app` and setup a Nuxt project inside it.
+
+```bash
+npm create nuxt@latest nuxt-app
+```
+
+Enter the newly made directory:
+
+```bash
+cd nuxt-app
+```
+
+### Install `workflow`
+
+<Tabs items={['npm', 'pnpm', 'yarn', 'bun']}>
+  <Tab value="npm">
+    <CodeBlock>npm i workflow</CodeBlock>
+  </Tab>
+  <Tab value="pnpm">
+    <CodeBlock>pnpm i workflow</CodeBlock>
+  </Tab>
+  <Tab value="yarn">
+    <CodeBlock>yarn add workflow</CodeBlock>
+  </Tab>
+  <Tab value="bun">
+    <CodeBlock>bun add workflow</CodeBlock>
+  </Tab>
+</Tabs>
+
+### Configure Nuxt
+
+Add `workflow/nuxt` to your `nuxt.config.ts`. This automatically configures the `workflow/nitro` module and enables usage of the `"use workflow"` and `"use step"` directives.
+
+```typescript title="nuxt.config.ts" lineNumbers
+import { defineNuxtConfig } from "nuxt/config";
+
+export default defineNuxtConfig({
+  modules: ["workflow/nuxt"], // [!code highlight]
+  compatibilityDate: "latest",
+});
+```
+
+<Accordion type="single" collapsible>
+  <AccordionItem value="typescript-intellisense" className="[&_h3]:my-0">
+    <AccordionTrigger className="[&_p]:my-0 text-lg [&_p]:text-foreground">
+      Setup IntelliSense for TypeScript (Optional)
+    </AccordionTrigger>
+    <AccordionContent className="[&_p]:my-2">
+To enable helpful hints in your IDE, setup the workflow plugin in `tsconfig.json`:
```

Comment:
I think we should do this automatically in the nuxt module in our generated `tsconfig.json` rather than require the user to do it.

if necessary we can have an option to disable